dns_filter.rst

```diff
@@ -52,11 +52,13 @@ All listed IP addresses or networks here will not be affected by DNS filter.
 Block certain websites
 -------------------------
 
-.. highlight:: bash
-
 .. _block_website-section:
 
-Access to certain websites can also be restricted by manipulating their DNS resolution, enabling ``AdBlock`` and, optionally, activating the DNS query interception feature for LAN clients.
+If you need to block specific domains and FQDNs you can do it directly from the Flashstart configuration page.
```

Comment:
Misspelled FlashStart. I'd also a link to the page of the manual.
```suggestion
If you need to block specific domains and FQDNs you can do it directly from the FlashStart configuration page.
```

Reply:
Nothing in contrary , but in this document (dns_filter) flashstart is always written this way: "Flashstart".
If we change it here we need to change also in other places.

Reply:
I'd say yes: we need to change it everywhere in the document.

Reply:
I changed them and added a reference to find the right place in flashstart panel where to write them.